### PR TITLE
Refine Ingest Data into RocketMQ

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

File: en_US/rule/bridge_rocketmq.md [hunk 1]
```diff
@@ -1,76 +1,67 @@
...
+Add action:
 
-在 “响应动作” 界面选择 “添加”，然后在 “动作” 下拉框里选择 “桥接数据到 RocketMQ”。
+In the **Action** pane, click **Add action**. Select **Data bridge to RocketMQ** from the drop-down list.
 
-![image](./assets/rule-engine/rocket-action-0@2x.png)
+<img src="./assets/rule-engine/rocketmq_action_type.png" alt="rocketmq_action_type" style="zoom:50%;" />
 
-填写动作参数:
+Provide the arguments for the action:
```
Comment (LenaLenaPan) [inline en_US/rule/bridge_rocketmq.md]:
```suggestion
You then need to specify the resource and topic that the rule applies :
```

**Review by LenaLenaPan — COMMENTED** [review]
add some comments

**Review by LenaLenaPan — DISMISSED** [review]
add some comments

**Review by wivwiv — DISMISSED** [review]

File: en_US/rule/bridge_rocketmq.md [hunk 2]
```diff
@@ -1,76 +1,67 @@
-# 桥接数据到 RocketMQ
+# Ingest Data into RocketMQ
```
Comment (LenaLenaPan) [inline en_US/rule/bridge_rocketmq.md]:
```suggestion
# Bridge Data into RocketMQ
```

**Review by LenaLenaPan — COMMENTED** [review]
add some comments

**Review by LenaLenaPan — APPROVED** [review]

**Pull request merged**